tests/watchdog_test.go

```diff
@@ -49,7 +49,22 @@ var _ = Describe("[sig-compute]Watchdog", decorators.SigCompute, func() {
 			By("Expecting the VirtualMachineInstance console")
 			Expect(console.LoginToAlpine(vmi)).To(Succeed())
 
-			By("Killing the watchdog device")
+			// Watchdog module need to be loaded as it will not be loaded by default for s390x
```

Comment:
i think ,we need to add  WgS390x label to run these test on s390x cluster after  merge